internal/restorer/filerestorer_test.go

```diff
@@ -56,10 +52,10 @@ func (i *TestRepo) packID(name string) restic.ID {
 	return i.packsNameToID[name]
 }
 
-func (i *TestRepo) pack(queue *packQueue, name string) *packInfo {
-	id := i.packsNameToID[name]
-	return queue.packs[id]
-}
+// func (i *TestRepo) pack(queue *packQueue, name string) *packInfo {
+// 	id := i.packsNameToID[name]
+// 	return queue.packs[id]
+// }
```

Comment:
Should we remove this part of dead code to keep it all clean? Or is there some specific reason to keep it?